README.md

```diff
@@ -54,7 +192,7 @@ doxygen .doxygen/Doxyfile
 
 the process outputs HTML and LaTeX documentations in `doc/html` and `doc/latex` folders.
 
-## Tests & Continuous Integration
+## Continuous Integration
```

Comment:
I think this section could be reworded --> we should explain that we have now the workflow "rules enforcer" and what is verified in this workfow...

Reply:
Maybe can be done in another PR ?

Reply:
ok